master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -1078,6 +1141,15 @@ private[celeborn] class Master(
     }
   }
 
+  override def getWorkerEventInfo(): String = {
+    val sb = new StringBuilder
+    sb.append("======================= Workers Event in Master ========================\n")
+    statusSystem.workerEventInfos.asScala.foreach { case (worker, workerEventInfo) =>
+      sb.append(s"${worker.toUniqueId().padTo(50, " ").mkString}$workerEventInfo\n")
+    }
+    sb.toString()
+  }
```

Comment:
Maybe we can add a parameter to filter if we only want some worker's event info?
Also we can add a rest info to show each worker's status?

Reply:
Good idea, Let's do this in next pr.